0.Preprocess_Data/scripts/2.preprocessing_nELISA_data.py

```diff
@@ -29,22 +31,21 @@
 )
 
 preprocessing_path = pathlib.Path(
-    f"../2.Nomic_nELISA_Analysis/Data/clean/Plate2/nELISA_plate_430420_{cell_type}_clean.csv"
+    f"../2.Nomic_nELISA_Analysis/Data/clean/Plate2/nELISA_plate_430420_{cell_type}_clean.parquet"
```

Comment:
I would recommend using `pathlib.path.resolve(strict=True)` just to capture the error here rather than anywhere else in the code.

Reply:
Addressed and fixed